templates/app/kafka-secrets.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kafka-secrets
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: kafka-secrets
+    labels:
+      app: "${APP_NAME}"
+  stringData:
+    hostname: "${QUEUE_HOSTNAME}"
+    password: "${QUEUE_PASSWORD}"
+    username: "${QUEUE_USERNAME}"
```

Comment:
Do these and the other QUEUE_ variables here and in other templates need to change to MESSAGING_?

Reply:
Sorry, didn't catch these. Technically we could have one name for the parameters and another for the ENV in the orchestrator, but it's probably better to be consistent.